sql_generators/active_users_aggregates_v3/templates/desktop_query.sql

```diff
@@ -7,7 +7,7 @@ WITH todays_metrics AS (
     app_version AS app_version,
     normalized_channel AS channel,
     IFNULL(country, '??') country,
```

Comment:
nitpicky, but for consistency:
```suggestion
    IFNULL(country, '??') AS country,
```

Reply:
Thanks!